### DISCUSSION TITLE
I want to create a topbar/Tabs view to manage multiple resources

### DISCUSSION THREAD

**Discussion by shym2501** [filamentphp/filament#11234] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I want to create a topbar/Tabs view as below: (Screenshot from http://demo.test/shop/customers/1/edit) But I don't want to create a "RelationsManager"

![image](https://github.com/filamentphp/filament/assets/53323434/8ec58924-d159-4d2f-9b55-0c7957ce5436)

Can I get some suggestions to make it? Or is there any plugin to help me?

I mean, I will make a navigation bar to navigate multiple "Resources". I think it's like "Clusters" but I want it to be TopBar/Tabs so it doesn't take up too much space.

**Comment by leandrocfe** [comment]
Do you mean, multiple tabs and each tab with a resource?

**Comment by e2d2-dev** [comment, marked as answer]
Hi,

just
- create a cluster (php artisan make:filament-cluster)
- add the resources you already have to it. ( protected static ?string $cluster = MyCluster::class; )
- add this ( protected static SubNavigationPosition $subNavigationPosition = SubNavigationPosition::Top; ) to each Resource to manipulate the position of the subNavigation

cheers

**Reply by shym2501** [reply]
Thanks for the answer. This is the solution I was looking for

I have implemented it and made some changes to the vendor file

![Web capture_6-2-2024_0400_filapos test](https://github.com/filamentphp/filament/assets/53323434/24649ae5-cee0-4c12-a503-9b69b8b7bfa9)

**Reply by SupianIDz** [reply]
Hi @shym2501 How do I make a tab cluster above the breadcrumbs and full width like you?

When I created it, the tabs were below the breadcrumbs and not full width
<img width="1438" alt="Screenshot 2024-02-17 at 05 53 45" src="https://github.com/filamentphp/filament/assets/37969970/03d22dbc-b0fe-4000-85bc-52eaa28f09c5">

**Reply by kmaphane** [reply]
Thank you for this gem. Tried to find it in the documents, with no luck. But it's up to all of us to contribute to the docs to make them better. thanks again though.

**Reply by e2d2-dev** [reply]
It's in the docs [https://filamentphp.com/docs/3.x/panels/clusters#creating-a-cluster](url), but on this kind of customizable package it's sometimes a PitA to find all the gems. Not everything is just related to the package, sometimes it's the underlaying stuff.

cheers

**Reply by shym2501** [reply]
> Hi @shym2501 How do I make a tab cluster above the breadcrumbs and full width like you?
> 
> When I created it, the tabs were below the breadcrumbs and not full width <img alt="Screenshot 2024-02-17 at 05 53 45" width="1438" src="https://private-user-images.githubusercontent.com/37969970/305566202-03d22dbc-b0fe-4000-85bc-52eaa28f09c5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY0ODEzNTYsIm5iZiI6MTcyNjQ4MTA1NiwicGF0aCI6Ii8zNzk2OTk3MC8zMDU1NjYyMDItMDNkMjJkYmMtYjBmZS00MDAwLTg1YmMtNTJlYWEyOGYwOWM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE2VDEwMDQxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhZWMxZmYxZjhkZTY4NWFlZDg4Njg1Y2RkM2QwMGQwMjQwMDcyMzgwYzc0ZDFmOWVmMzlkMzBlNTUwNjg1MGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Ku2yhl7zBZFahVHxMCH752IZdJZXJmq9-mqfJygRA5I">

I made it manually by changing some css files from the filament source code

**Reply by shym2501** [reply]
> Thank you for this gem. Tried to find it in the documents, with no luck. But it's up to all of us to contribute to the docs to make them better. thanks again though.

You can try this: https://filamentphp.com/docs/3.x/panels/resources/getting-started#sub-navigation-position
